flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecMLPredictTableFunction.java

```diff
@@ -81,8 +81,8 @@
         name = "stream-exec-ml-predict-table-function",
         version = 1,
         producedTransformations = StreamExecMLPredictTableFunction.ML_PREDICT_TRANSFORMATION,
-        minPlanVersion = FlinkVersion.V2_1,
```

Comment:
Do we need to modify this?

Reply:
It's fixed in https://github.com/apache/flink/pull/26661. I may need to rebase